codersdk/rbacresources.go

```diff
@@ -27,6 +27,47 @@ const (
 	ResourceSystem                      RBACResource = "system"
 )
 
+const (
+	ActionCreate = "create"
+	ActionRead   = "read"
+	ActionUpdate = "update"
+	ActionDelete = "delete"
+)
+
+var (
+	AllRBACResources = []RBACResource{
```

Comment:
nit: It looks like something devs may forget to update in the future.

Reply:
We have rbac gen: https://github.com/coder/coder/blob/4463012b6d91e2bc46e066b4206e9eeb84d04668/coderd/rbac/object_gen.go#L1-L1

I wonder if we can repurpose it for this too? 

In a more general note though, we manually copy over all enums we have in the database package to the sdk package. So this problem already exists for all enum types.